fdbclient/BackupContainerAzureBlobStore.actor.cpp

```diff
@@ -167,8 +168,13 @@ class BackupContainerAzureBlobStoreImpl {
 		if (!exists) {
 			throw file_not_found();
 		}
-		return Reference<IAsyncFile>(
-		    new ReadFile(self->asyncTaskThread, self->containerName, fileName, self->client.get()));
+		Reference<IAsyncFile> f =
+		    makeReference<ReadFile>(self->asyncTaskThread, self->containerName, fileName, self->client.get());
+		if (self->usesEncryption()) {
+			f = makeReference<AsyncFileEncrypted>(f, false);
+		}
+		f = makeReference<ReadFile>(self->asyncTaskThread, self->containerName, fileName, self->client.get());
```

Comment:
Maybe I'm misunderstanding how this works, but isn't this overwriting the `f` from the block above? It doesn't seem to reference the encrypted `f` at all, whereas most other places where this pattern occurs seems to contain `f` in the `makeReference` call

Reply:
Good catch, I will remove this line.